police_request/PoliceRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+    
+    /**
+     * Example code (commented) at the bottom of this file.
+     * If you have any problems, open an issue on GitHub or
+     * email me at cameron.white@ufl.edu
```

Comment:
You might want to include a non-ufl email here as well, mine was deactivated when I graduated.